### PR TITLE
Request Body None feature

### PR DESCRIPTION
**Pull request by ArifPMan**
Documented the feature. Please review and approve.

### CODE REVIEW

**Comment by ArifPMan** [comment]
Made the changes. Harry please review and approve the PR.

File: v6/postman/sending_api_requests/requests.md [hunk 1]
```diff
@@ -96,10 +96,16 @@ Use the control dropdown to change the request method. The request body editor a
...
+While constructing requests, you'll work frequently with the request body editor. Postman lets you send almost any kind of HTTP request. The body editor is divided into five areas and has different controls, depending on the body type.
 
 **Note about Headers**: When you are sending requests through the HTTP protocol, your server might expect a Content-Type header. The Content-Type header allows the server to parse the body properly. For form-data and urlencoded body types, Postman automatically attaches the correct Content-Type header so you don't have to set it. The raw mode header is set when you select the formatting type. If you manually use a Content-Type header, that value takes precedence over what Postman sets. Postman does not set any header type for the binary body type.
 
+##### None
+
+ Postman creates a body by default and 'none' radio button is selected, irrespective of the fact that your method has a body or not. If your method has a body, you may choose an appropriate option available there. The default body option appears as below:
```
[the remark this answers is not in the document]
Comment (ArifPMan) [reply v6/postman/sending_api_requests/requests.md]: Updated with the f/b comments.

File: v6/postman/sending_api_requests/requests.md [hunk 2]
```diff
@@ -96,10 +96,16 @@ Use the control dropdown to change the request method. The request body editor a
...
+While constructing requests, you'll work frequently with the request body editor. Postman lets you send almost any kind of HTTP request. The body editor is divided into five areas and has different controls, depending on the body type.
 
 **Note about Headers**: When you are sending requests through the HTTP protocol, your server might expect a Content-Type header. The Content-Type header allows the server to parse the body properly. For form-data and urlencoded body types, Postman automatically attaches the correct Content-Type header so you don't have to set it. The raw mode header is set when you select the formatting type. If you manually use a Content-Type header, that value takes precedence over what Postman sets. Postman does not set any header type for the binary body type.
 
+##### None
+
+ Postman defaults to 'none' radio button whenever you send a request method, irrespective of the fact that your request has a body or not. Choose this option if you don't want to send body. If your request has a body, you may choose an appropriate option available there. The default body option is none and it appears as below:
```
Comment (numaanashraf) [inline v6/postman/sending_api_requests/requests.md]: "The default body option is none" this is already mentioned in the first paragraph - "Postman defaults to 'none' "...

Comment (numaanashraf) [inline v6/postman/sending_api_requests/requests.md]: 'none' is not a radio button. its a body type.

File: v6/postman/sending_api_requests/requests.md [hunk 3]
```diff
@@ -96,10 +96,16 @@ Use the control dropdown to change the request method. The request body editor a
 
 ### Request Body
 
-While constructing requests, you'll work frequently with the request body editor. Postman lets you send almost any kind of HTTP request. The body editor is divided into 4 areas and has different controls, depending on the body type.
+While constructing requests, you'll work frequently with the request body editor. Postman lets you send almost any kind of HTTP request. The body editor is divided into five areas and has different controls, depending on the body type.
 
 **Note about Headers**: When you are sending requests through the HTTP protocol, your server might expect a Content-Type header. The Content-Type header allows the server to parse the body properly. For form-data and urlencoded body types, Postman automatically attaches the correct Content-Type header so you don't have to set it. The raw mode header is set when you select the formatting type. If you manually use a Content-Type header, that value takes precedence over what Postman sets. Postman does not set any header type for the binary body type.
 
+##### None
```
Comment (numaanashraf) [inline v6/postman/sending_api_requests/requests.md]: "irrespective of the fact that your request has a body or not". not sure why we need this. whether a request has a body or not is up to the user, and this option sets body to none.

File: v6/postman/sending_api_requests/requests.md (hunk 2 above)
Comment (ArifPMan) [reply v6/postman/sending_api_requests/requests.md]: Updated with your f/b. Please review

File: v6/postman/sending_api_requests/requests.md [hunk 4]
```diff
@@ -96,10 +96,16 @@ Use the control dropdown to change the request method. The request body editor a
...
+
+ Postman defaults to 'none' option whenever you send a request method. Choose this option if you don't want to send body with your request. If your request has a body, you may choose an appropriate option available there. The default body option appears as below:
```
Comment (harryi3t) [inline v6/postman/sending_api_requests/requests.md]: `... whenever you send a request. Choose ...`

Comment (ArifPMan) [reply v6/postman/sending_api_requests/requests.md]: Made the change.

**Pull request merged**